test/end-to-end/thirdpartyemailpassword.test.js

```diff
@@ -141,6 +144,28 @@ describe("SuperTokens Third Party Email Password", function () {
             assert.deepStrictEqual(pathname, "/dashboard");
         });
 
+        it("should show error message on sign up with duplicate email", async function () {
+            await toggleSignInSignUp(page);
+
+            // Clear cookies, try to signup with same address.
+            consoleLogs = await clearBrowserCookiesWithoutAffectingConsole(page, consoleLogs);
+            await page.goto(`${TEST_CLIENT_BASE_URL}/auth`, {
+                waitUntil: "networkidle0",
+            });
+            await toggleSignInSignUp(page);
+
+            // Set values.
+            const [, hasEmailExistMethodBeenCalled] = await Promise.all([
+                setInputValues(page, [{ name: "email", value: "john.doe@supertokens.io" }]),
+                hasMethodBeenCalled(page, EMAIL_EXISTS_API),
+            ]);
+
+            // Assert.
+            assert.strictEqual(hasEmailExistMethodBeenCalled, false);
+            let formFieldErrors = await getFieldErrors(page);
+            assert.deepStrictEqual(formFieldErrors, ["This email already exists. Please sign in instead"]);
```

Comment:
there is another variation of this test in which we want to call the sign up API and have that show up the error for "This email already exists. Please sign in instead". This can be done by overriding the doesEmailExist recipe function to always return false and not query the API.

Reply:
okay, do you want to do it this way or you are just letting me know ?

Reply:
What do you mean? You need to write the test which actually calls the sign up API and makes sure that the form field error from it is properly shown in the UI

Reply:
apologies, misunderstood